cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/testdata/gcloud-setup.md

```diff
@@ -8,20 +8,15 @@ The following service account was set up for use by the integration tests using
 Google Cloud Console
 - GCP project: `vmware-kubeapps-ci`
 - GCP service account:
-    - Name and ID: `flux-plugin-test-sa-3`
+    - Name and ID: `flux-plugin-test-sa-4`
     - Description: `Service Account for integration testing of kubeapps flux plugin`
 
   Roles granted to this service account:
-    - Editor. Verified 8/24/22 11:39pm this works. 
-
-  **TODO** need to reduce permissions to smallest workable set. 
-  Probably some combination of these and others:
     - Artifact Registry Administrator
     - Artifact Registry Repository Administrator
-    - Artifact Registry Service Agent
+    - Viewer
 
-  Make sure you see a message "Policy Updated" at the bottom of the screen when you grant these roles.
-  If you see "Failed to add project roles" or some other error message,
+  Make sure you see a message "Policy Updated" at the bottom of the screen when you grant these roles. If you see "Failed to add project roles" or some other error message,
   create the service account with a different Name/ID
```

Comment:
Just out of curiosity: is this why `flux-plugin-test-sa-*` is increasing the number?
Do you know the problem behind?

Reply:
No technical reason. There are times when I have had multiple service accounts active for testing, so I increment the Name/ID to keep track of what's what. But I always make sure to clean it up and the end so there is a single service account left
<img width="1393" alt="Screen Shot 2022-08-31 at 9 47 51 AM" src="https://user-images.githubusercontent.com/74032303/187734287-ca03a82e-5d1f-4868-a86f-f07e5bbdbb9f.png">